thesis-api-v2/src/main/java/pl/edu/agh/quizzesthesis/business/service/UnsplashApiService.java

```diff
@@ -41,23 +41,28 @@ private void getFromUnsplash(Term term) {
         var fromResponse = getFromResponse(responseBody);
         term.setPictureURL(fromResponse.pictureURL());
         term.setAuthorName(fromResponse.authorName());
+        term.setAuthorProfileURL(fromResponse.authorProfileURL());
     }
 
     @NotNull
     private ResponseSpec getResponseSpec(String termName) {
+        if (env.getProperty("UNSPLASH_API_KEY") == null || env.getProperty("UNSPLASH_API_KEY").isEmpty()) {
```

Comment:
I would extract "UNSPLASH_API_KEY" to a private static final field.

Reply:
Nice catch, generally I will leave this MR open since our approval was rejected? Either way I will probably implement an endpoint on backend site that would send a request to "trigger download" (honestly I wonder if just sending some curls on this "download endpoint" just to show them that we know how to implement it would not be better, since we will not use it after all).